packages/hash/frontend/src/blocks/page/Comments/CommentBlock.tsx

```diff
@@ -0,0 +1,186 @@
+import { FunctionComponent, useMemo, useState, useRef, useEffect } from "react";
+import { Box, Typography } from "@mui/material";
+import {
+  Avatar,
+  Button,
+  FontAwesomeIcon,
+  IconButton,
+} from "@hashintel/hash-design-system/ui";
+import { formatDistanceToNowStrict } from "date-fns";
+import {
+  faChevronDown,
+  faChevronUp,
+  faEllipsisVertical,
+  IconDefinition,
+} from "@fortawesome/free-solid-svg-icons";
+import { usePopupState } from "material-ui-popup-state/hooks";
+import { bindTrigger } from "material-ui-popup-state";
+import { types } from "@hashintel/hash-shared/types";
+import { extractBaseUri } from "@blockprotocol/type-system-web";
+import { PageComment } from "../../../components/hooks/usePageComments";
+import { CommentTextField, CommentTextFieldRef } from "./CommentTextField";
+import { CommentBlockMenu } from "./CommentBlockMenu";
+import styles from "./style.module.css";
+
+const LINE_HEIGHT = 21;
+
+type ShowMoreTextLinkProps = {
+  label: string;
```

Comment:
This is more accurate 

```suggestion
  label: ReactNode;
```

Reply:
Changed in 77ecdbfbc2f1dff5e8952317e4e6f105b93a652e.